CoreAttributeEditorView/src/au/gov/asd/tac/constellation/views/attributeeditor/AttributeEditorPanel.java

```diff
@@ -237,10 +238,44 @@ public AttributeEditorPanel(final AttributeEditorTopComponent parent) {
             root.getChildren().add(tooltipPane);
             this.setCenter(root);
         });
-
+        showEmptyToggles = new HashMap<HeadingType, ToggleButton>();
         updateEditorPanel(null);
     }
 
+    /**
+     * Every time that the 'Attribute Editor' is redisplayed, reset the
+     * 'Show Empty' toggles to be toggled on.
+     */
+    public void refreshShowEmpty()
+    {
```

Comment:
are you able to put the open braces at the end of the previous line (I think there is a few places this can be done)